CONTRIBUTING.md

```diff
@@ -206,24 +198,35 @@ The following checklist is used when reviewing pull requests
 ### General (all submissions)
 
 - Commit messages are useful and properly formatted
+- No sensitive personally identifiable information (PII) is included
```

Comment:
@xiangce No sensitive personally identifiable information (PII) is included in any types of files nor docstring

Reply:
Done